packages/repocop/src/rules/repository.ts

```diff
@@ -269,49 +270,78 @@ function findArchivedReposWithStacks(
 export async function getAlertsForRepo(
 	octokit: Octokit,
 	name: string,
-): Promise<Alert[] | undefined> {
+): Promise<PartialAlert[] | undefined> {
 	if (name.startsWith('guardian/')) {
 		name = name.replace('guardian/', '');
 	}
 
 	try {
-		const alert = await octokit.rest.dependabot.listAlertsForRepo({
-			owner: 'guardian',
-			repo: name,
-			per_page: 100,
-			severity: 'critical', //eventually this should be "critical,high"
-			state: 'open',
-		});
-
-		return alert.data;
+		const alert: DependabotVulnResponse =
+			await octokit.rest.dependabot.listAlertsForRepo({
+				owner: 'guardian',
+				repo: name,
+				per_page: 100,
+				severity: 'critical,high',
+				state: 'open',
+				sort: 'created',
+				direction: 'asc', //retrieve oldest vulnerabilities first
+			});
+
+		return alert.data.map((a) => a as PartialAlert);
```

Comment:
Probably don't need the `map` here?

```suggestion
		return alert.data as PartialAlert[];
```

Or potentially, given that the full and partial alert types overlap sufficiently, no assertion at all?

```suggestion
		return alert.data;
```